warehouse/templates/pages/help.html

```diff
@@ -3,7 +3,7 @@
  # you may not use this file except in compliance with the License.
  # You may obtain a copy of the License at
  #
- # http://www.apache.org/licenses/LICENSE-2.0
+ # //www.apache.org/licenses/LICENSE-2.0
```

Comment:
Ah, OK. I thought that it might be a bug in the linter, because I know it prefers links without protocols. Why did this change then?